other/bootstrap_daemon/README.md

```diff
@@ -70,3 +70,47 @@ sudo grep "tox-bootstrapd" /var/log/syslog
 - Make sure you have write permission for keys and pid files.
 
 - Make sure you have read permission for the config file.
+
+
+##Instructions for ArchLinux
+
+All commands needs to be run as a root user or via sudo.
+
+Create new user:
+```sh
+useradd --home-dir --shell /bin/false --user-group tox
+```
+
+Place tox-bootstrapd.conf into /etc/tox and customize for your needs:
+```sh
+cp tox-bootstrapd.conf /etc/tox/
+vim /etc/tox/tox-bootstrapd.conf
+# make sure, that you set the followings paths:
+#   keys_file_path = "/etc/tox/key"
+#   pid_file_path = "/etc/tox/tox-bootstrapd.pid"
```

Comment:
[PID files should be stored in /var/run.](http://tldp.org/LDP/Linux-Filesystem-Hierarchy/html/Linux-Filesystem-Hierarchy.html#var)